pkg/models/const.go

```diff
@@ -23,6 +23,7 @@ var orangeColorSGR = []color.Attribute{38, 5, 208}
 var HighlightString = color.New(orangeColorSGR...).SprintFunc()
 var HighlightPassingString = color.New(color.FgGreen).SprintFunc()
 var HighlightFailingString = color.New(color.FgRed).SprintFunc()
+var GrayString = color.New(color.FgHiBlack)
```

Comment:
Just a small thing, can you please change GrayString to HighlightStringGray = color.New(color.FgHiBlack).SprintFunc()
and also change the affected parts of the code.